modules/apps/portal-workflow/portal-workflow-web/src/test/java/com/liferay/portal/workflow/web/internal/util/filter/WorkflowDefinitionImpl.java

```diff
@@ -100,6 +102,11 @@ public int getVersion() {
 		return 0;
 	}
 
+	@Override
+	public List<WorkflowTransition> getWorkflowTransitions() {
+		return null;
+	}
+
```

Comment:
@MarinhoFeliphe this change should be part of this commit https://github.com/liferay-workflow/liferay-portal/pull/466/commits/7d47b20d4fd5ada6e258e768bd32022e090880df instead.